helix-ui/README.md

```diff
@@ -0,0 +1,77 @@
+# helix-ui
+
+After building the project from the root directory (i.e. `./build`), find the
+`helix-ui-${version}.jar` artifact in this module's `target` directory.
+
+To run the UI server in read-only mode with no configuration, execute the
+following commands:
+
+```
+java -jar helix-ui-${version}.jar server
+```
+
+Navigate to `http://localhost:8080/dashboard` to get started. At this page,
+enter a ZooKeeper address, e.g. "localhost:2181,localhost:2182" or
+"some-machine:2181/chroot", to get started.
+
+The following shows using the dashboard to view the end state of MyResource in
+the Quick Start, in tabular form:
+
+![Quick Start End Table](doc/quickstart-end-table.png)
```

Comment:
If a resource is in FULL_AUTO, you'd probably see the instance name changing in the tabular view for each resource, and you'd probably see the partitions move among the instance circles in the graphical view.